mytools.py

```diff
@@ -225,10 +225,14 @@ def open_video_writer(fname, w, h, fps=30):
     w, h - frame width/height
     """
 
+    codec = (
+        cv2.VideoWriter_fourcc("v", "p", "0", "9")
```

Comment:
If I recall correctly, the issue with mp4v was related to the codec not being readily available on linux, it might be worthwhile to keep vp09 for that platform, maybe as a note for future development

Reply:
On Windows, this codec works SOO SLOOOW, that frame rate drops to 1-2 FPS vs 90 FPS when using mp4v. That was the reason to leave it only for tests. I will try to do soft fallback though...

Reply:
it's probably worth investigating further, maybe something like vp08 is a better match, my criteria for selecting it were that it works on windows/linux with just the requirements and that it displays in browser